experiment/test_measurer.py

```diff
@@ -431,12 +441,24 @@ def test_measure_loop_end(_, __, ___, ____, _____, ______, experiment_config,
                           db_experiment):
     """Tests that measure_loop stops when there is nothing left to measure. In
     this test, there is nothing left to measure on the first call."""
-    measurer.measure_loop(experiment_config, 100)
+
+    # Register timeout handler and set timeout.
```

Comment:
The measure loop catches all exceptions *inside* the loop. During development, this test (to check whether the measurer is always terminating) froze because of a handled exception. The signal is simply a timeout to handle the freezing loop. We can add a comment or remove the code.

Reply:
@mboehme, I removed it already. Just making few more changes before I commit.